imports/ui/components/Footer.jsx

```diff
@@ -9,6 +10,7 @@ class Footer extends Component {
           <span>&copy; 2017 All Rights Reserved</span>
           <span>United Way of Central Indiana</span>
           <span>Website by Indy Tech Fellowship</span>
+          <Link to={'/termsandconditions'}>Terms and Conditions</Link>
```

Comment:
were you going to use the browseHistory.push() here onClick instead of the Link?